README.md

```diff
@@ -1,28 +1,56 @@
 ## Ember Guides Source
 
 [![Build Status](https://travis-ci.org/emberjs/guides.svg?branch=master)](https://travis-ci.org/emberjs/guides)
-[![Crowdin](https://d322cqt584bo4o.cloudfront.net/emberjs/localized.svg)](https://crowdin.com/project/emberjs)
 
-The source for the Ember.js Guides.
+This is the source for the [Ember.js Guides](https://guides.emberjs.com).
 
-## Contributing
-
-Please see [CONTRIBUTING.md](CONTRIBUTING.md).
+Looking for repositories for other parts of the site? Check out
+[website](https://github.com/emberjs/website),
+[ember-api-docs](https://github.com/ember-learn/ember-api-docs),
+[super-rentals tutorial](https://github.com/ember-learn/super-rentals),
+[statusboard](https://github.com/ember-learn/statusboard),
+and [styleguide](https://github.com/ember-learn/ember-styleguide)
 
-## Translations
+## Contributing
 
-Please see [TRANSLATIONS.md](TRANSLATIONS.md).
+Welcome and thanks for your help! Please see [CONTRIBUTING.md](CONTRIBUTING.md)
+for detailed instructions on how to format your work and submit a Pull Request.
 
 ## Project layout
 
-The Guides' content is in the `source` folder. The left nav bar is produced from
+The Guides content takes the form of Markdown files (just like most READMEs).
+The Guides themselves are in the `source` folder. The left nav bar is produced from
 `data/pages.yml`. `lib` contains Middleman plugins, and `spec` contains tests
 for those plugins.
 
-## Dependencies
+## Running locally with Docker (recommended)
+
+This is the recommended method for new contributors.
+Although the Guides are built with Ruby, most work is done in Markdown files.
+You don't need to know Ruby or install its dependencies to help out. Simply follow
+the Docker container instructions below to install and run locally.
 
-The Guides are built with Middleman, which runs on Ruby 1.9.3 or newer
-(2.0.0 recommended).
+First, install [Docker](https://www.docker.com/) and leave it running.
```

Comment:
this should probably be [Docker](https://docs.docker.com/engine/installation/) and [Docker Compose](https://docs.docker.com/compose/install/)
not sure if you want to link directly to the appropriate docs or not -- not sure how often the urls change.